tfjs-data/src/iterators/microphone_iterator_test.ts

```diff
@@ -214,11 +216,11 @@ describeBrowserEnvs('MicrophoneIterator', () => {
     // Call iterator.next() every 200 milliseconds, stop after 3 times.
     const interval = setInterval(getTensor, 200);
 
-    // Wait 4 seconds for the intervals to run.
+    // Wait 3 seconds for the intervals to run.
     await new Promise(resolve => {
```

Comment:
is it possible to write these tests in such a way we don't actually have to wait 3 seconds? like mocking timing information so this is a quicker test?

Reply:
Mocking timing information of setInterval is quite tricky. I updated microphone configs in tests to reduce the time to produce a tensor. Each test should only take several milliseconds now.